extensions/adobe/experience/customerJourneyManagement/messageexecution.schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/customerJourneyManagement/messageexecution",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe CJM ExperienceEvent - MessageExecution Details",
+  "description": "MessageExecution details for the Adobe CJM ExperienceEvent.",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+
+  "definitions": {
+    "messageExecution": {
+      "properties": {
+        "@id": {
```

Comment:
@cmathis : As per the https://github.com/adobe/xdm/blob/master/CONTRIBUTING.md we should not be inventing the new "id" attributes, but to use the "@id" convention.

Can you help usunderstand, how this "@id" field is namespaced, we may have a case where we add more than one mixin to schema, where both the mixjns has "@id" field. Then how this two "@id" fields are distinguished.

cc: @aashishjassal

Reply:
The ExperienceEvent already includes an @id field, so this is basically ignored here. If you need an execution Id or something like it, then create a new field here named "executionId".

Reply:
I have replaced the "@id" field  with the full name spaced field name.